pg_search/src/postgres/customscan/pdbscan/projections/snippet.rs

```diff
@@ -194,7 +197,7 @@ unsafe fn extract_snippet_text(
     {
         let attname = (*context)
             .attname_lookup
-            .get(&((*context).rti as _, (*field_arg).varattno as _))
+            .get(&((*context).planning_rti as _, (*field_arg).varattno as _))
```

Comment:
I'm confused...

Reply:
Can we add an assert somewhere in both of these functions that the var we are looking us is actually from the `planning_rti`?

Reply:
As we discussed offline, I ran the whole test-suite with this assertion and there's no issue detected.